lib/ansible/modules/system/sysctl.py

```diff
@@ -211,12 +211,17 @@ def _values_is_equal(self, a, b):
     def _parse_value(self, value):
         if value is None:
             return ''
-        elif isinstance(value, bool):
+        if isinstance(value, bool):
             if value:
                 return '1'
             else:
                 return '0'
-        elif isinstance(value, basestring):
+        # Check for python 2.x basestring which unifies (unicode, str) support
+        try:
+            basestring
+        except NameError:
+            basestring = str
+        if isinstance(value, basestring):
```

Comment:
`ansible.module_utils.basic` already declares `basestring` this way. If the import in this module were updated to include the variable or `*` that would also address the issue.

Ref:
https://github.com/ansible/ansible/blob/631a10745d344c85817ab504f678d2c40ec8ae4b/lib/ansible/module_utils/basic.py#L195-L200